src/plugins/intel_npu/src/plugin/npuw/llm_compiled_model.cpp

```diff
@@ -301,6 +301,11 @@ struct NPUDesc {
 };
 
 std::optional<NPUDesc> extract_npu_descriptor(const std::shared_ptr<const ov::IPlugin>& plugin) {
+    const auto all_devices = plugin->get_core()->get_available_devices();
+    if (std::find(all_devices.begin(), all_devices.end(), "NPU") == all_devices.end()) {
```

Comment:
@AsyaPronina could you please use `get_core()->get_property("NPU", ov::available_devices)` instead ?

It will temporary help with CVS-161330

CC @sshlyapn

Reply:
PR https://github.com/openvinotoolkit/openvino/pull/28730